executor/src/witgen/machines/mod.rs

```diff
@@ -350,7 +348,43 @@ impl<T: Display> Display for Connection<'_, T> {
     }
 }
 
-impl<T> Connection<'_, T> {
+impl<'a, T: FieldElement> Connection<'a, T> {
+    /// Creates a connection if the identity is a bus send.
```

Comment:
So all connections are derived from bus sends?

Reply:
Yes, because RN the assumption is still that you statically find the receive. But this already allows for many-to-one connections!